featurebyte/models/offline_store_feature_table.py

```diff
@@ -43,6 +43,17 @@ class OnlineStoreLastMaterializedAt(BaseModel):
     value: datetime
 
 
+class PrecomputedLookupFeatureTableInfo(BaseModel):
+    """
+    Metadata for a feature table that is derived from a source feature table in order to support
+    precomputed lookup using a related entity
+    """
+
+    lookup_steps: List[EntityRelationshipInfo]
+    feature_list_ids: List[PydanticObjectId] = Field(default_factory=list)
```

Comment:
Do we need the feature list ids for materialization of these derived feature tables?

Reply:
Feature list ids are not needed for materialization, but needed for feast registry construction (not included in this PR).